ShellPkg/Library/UefiShellDebug1CommandsLib/Pci.c

```diff
@@ -5452,7 +5482,7 @@ PrintInterpretedExtendedCompatibilityLinkControl (
   DumpHex (

```

Comment:
Wouldn't it be simpler to have a `CheckPciDumpHex()` wrapper as:
```
CheckPciDumpHex (
  IN UINTN  Indent,
  IN UINTN  Offset,
  IN UINTN  DataSize,
  IN VOID   *UserData
  )
{
  ...
  [Do some check]
  DumpHex(Indent, Offset, DataSize, UserData);
  ...
}
```
It would allow to avoid long lines + minimize the changes in this patch.
A better name for the function might be chosen